spec/policies/timelog_policy_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rails_helper'
+
+describe TimelogPolicy do 
+  subject { TimelogPolicy }
+
+  let (:current_user) { FactoryGirl.create :user }
+  let (:other_user) { FactoryGirl.create :user }
+  let (:admin) { FactoryGirl.create :user, role: 'admin' }
+  let (:user_timelog) { FactoryGirl.create :timelog, user_id: current_user.id }
+  let (:other_timelog) { FactoryGirl.create :timelog, user_id: other_user.id }
+
+  # Index action gets @timelogs as an array, so here we have
+  # to do a little trick with pushing to array, otherwise 
+  # policy will crash on method empty? for @timelogs 
+  permissions :index? do    
+    it 'allows user to access your timelogs' do
+      timelogs_arr = []
```

Comment:
ruby is beatiful language :). User human friendly var names. `timelogs` is much better than `timelogs_arr` (which is close to some C++, good example would be t_arr)